tests/approaches/test_nsrt_learning_approach.py

```diff
@@ -190,3 +191,13 @@ def test_grammar_search_invention_approach():
                    try_solving=False,
                    sampler_learner="random",
                    additional_settings=additional_settings)
+
+
+def test_sampler_learning_with_goals():
```

Comment:
verify speed of this test (`pytest tests/approaches/test_nsrt_learning_approach.py --durations=0`)

Reply:
I get around 10.4s for this test

Reply:
do you mean the speed of just this one function you added, or the entire `test_nsrt_learning_approach.py`?

Reply:
I meant the entire file; the one function added takes ~2.4s

Reply:
okay, not a problem then!